### PR TITLE
Reduce flake8 max-line-length from 146 to 145

### PR DESCRIPTION
**Pull request by ridaaysha070**
# Description

Reduced the max-line-length in flake8 from 146 to 145. Fixed the subsequent errors due to files exceeding the max-line-length.

# Changed Behaviour

# Fixes

Fixes #3105 

## Type of change

Choose which options apply, and delete the ones which do not apply.

- Code maintenance/cleanup

### CODE REVIEW

**Comment by ridaaysha070** [comment]
Hi @benclifford, I reduced the line length from 147 to 146 and fixed all the errors. I believe all the automated tests have also passed. Let me know if there is anything else I should add. Thanks!

File: parsl/monitoring/db_manager.py [hunk 1]
```diff
@@ -660,7 +662,9 @@ def _insert(self, table: str, messages: List[MonitoringMessage]) -> None:
                     done = True
                 except sa.exc.OperationalError as e:
                     # hoping that this is a database locked error during _update, not some other problem
-                    logger.warning("Got a database OperationalError. Ignoring and retrying on the assumption that it is recoverable: {}".format(e))
+                    logger.warning(
+                        f"Got a database OperationalError."
```
Comment (benclifford) [inline parsl/monitoring/db_manager.py:666]: Missing space at the end of this string - putting two `""`-strings next to each other doesn't put a space between them, but the original message did have a space.

File: parsl/monitoring/db_manager.py [hunk 2]
```diff
@@ -633,7 +633,9 @@ def _update(self, table: str, columns: List[str], messages: List[MonitoringMessa
                     # if retried - for example, the database being locked because someone else is readying
                     # the tables we are trying to write to. If that assumption is wrong, then this loop
                     # may go on forever.
-                    logger.warning("Got a database OperationalError. Ignoring and retrying on the assumption that it is recoverable: {}".format(e))
+                    logger.warning(
+                        f"Got a database OperationalError."
```
Comment (benclifford) [inline parsl/monitoring/db_manager.py:637]: missing space - see comment on line 665

**Review by benclifford — CHANGES_REQUESTED** [review]

**Comment by ridaaysha070** [comment]
@benclifford Hey, I've updated the PR to go from 146 to 145 and also fixed the spacing issues you mentioned above. Thanks again!

**Review by benclifford — APPROVED** [review]

**Pull request merged**